util/src/main/java/tech/pegasys/artemis/util/config/NetworkDefinition.java

```diff
@@ -24,14 +24,18 @@
 
 public class NetworkDefinition {
 
+  public static final String MINIMAL = "minimal";
+  public static final String MAINNET = "mainnet";
+  public static final String TOPAZ = "topaz";
+
   private static final ImmutableMap<String, NetworkDefinition> NETWORKS =
       ImmutableMap.<String, NetworkDefinition>builder()
-          .put("minimal", builder().constants("minimal").startupTargetPeerCount(0).build())
-          .put("mainnet", builder().constants("mainnet").build())
+          .put(MINIMAL, builder().constants(MINIMAL).startupTargetPeerCount(0).build())
```

Comment:
Could we inline these constants again?  `mainnet` network name and `mainnet` constants spec are different things (when MainNet actually exists it will have boot nodes, contract address etc).  We shouldn't use the same constant to refer to different things.  And generally this reads better and more succinctly with the inline string constants due to the improved locality of reference (ie you don't need to also look at the definition of the constant to work out what is actually being set).

Reply:
they're new constants on the NetworkOptions class, but i can remove them i guess. I'd really prefer not to use repeated literal strings in this way, it's asking for trouble.

Reply:
I'm really keen to avoid using the same constant for two different things.  Just because they are the same string, doesn't make them the same.